aptos-move/framework/aptos-framework/sources/multisig_account.move

```diff
@@ -413,6 +419,46 @@ module aptos_framework::multisig_account {
         );
     }
 
+    /// Creates a new multisig account on top of an existing account and immediately rotate the origin auth key to 0x0.
+    ///
+    /// Note: If the original account is a resource account, this does not revoke all control over it as if any
+    /// SignerCapability of the resource account still exists, it can still be used to generate the signer for the
```

Comment:
How is it possible? I think resource_account cannot generate create_multisig_account_signed_message because nobody has the private key?

Reply:
The resource account could have had the auth key set to an actual key (via the flow in resource_account.move) that they used to to create rotation or signer capability offers